src/commands.js

```diff
@@ -110,7 +110,8 @@ PhotoHandler = async (ctx) => {
     const caption = ctx.message.caption || ""
     const target = ctx.isAngel ? ctx.angel : ctx.mortal
     if (target.isRegistered()) {
-        const fileLink = await ctx.telegram.getFileLink(photos[0].file_id)
+        const len = ctx.message.photo.length - 1
```

Comment:
Oof please don't do this! Because it would be wrong to say that this is the length of photos. Instead, a better way could be
```
const len = ctx.message.photo.length
const fileLink = await ctx.telegram.getFileLink(photos[len-1].file_id)
```